.github/workflows/code_metrics_pr.yml

```
@@ -66,7 +68,7 @@ jobs:
       - name: Create index page
         run: |
           cd pr-check
-          tree -H 'https://tier4.github.io/obstacle_stop_planner_refine/pr-check' \
+          tree -H '$GITHUB_PAGES_URL/pr-check' \
```

Single quotes do not expand variables and must be changed to double quotes.

```suggestion
          tree -H "$GITHUB_PAGES_URL/pr-check" \
```